core/src/main/java/com/graphhopper/speeds/WaySpeedsProvider.java

```diff
@@ -0,0 +1,10 @@
+package com.graphhopper.speeds;
+
+import com.graphhopper.routing.ev.RoadClass;
+
+import java.util.Optional;
+
+public interface WaySpeedsProvider {
+    Optional<Double> speedForWay(long osmWayId);
```

Comment:
We are not modeling Units here, in that case, should we make it explicit in the name of the method?

`speedOnKmPerHourForWay`